### PR TITLE
Kubernetes Handbook articles

### PR DESCRIPTION
**Pull request by bryanfriedman**

### CODE REVIEW

**Comment by zmb3** [comment]
If we want to wait on this until #521 and #537 are merged, I'm happy to look at how to fold this content into the new subsections created in those other PRs.

**Comment by bryanfriedman** [comment]
> If we want to wait on this until #521 and #537 are merged, I'm happy to look at how to fold this content into the new subsections created in those other PRs.

Thanks Zac. I think I did okay with it. @BrianMMcClain is reviewing but feel free to take a look as well. Hoping to merge later today...

File: content/guides/kubernetes/from-docker-to-kubernetes.md
Comment (BrianMMcClain) [inline content/guides/kubernetes/from-docker-to-kubernetes.md:44]:
```suggestion
When you give this Dockerfile to a local instance of Docker by using the `docker build` command, it creates a container image with the "Hello World" app installed in it.
```

File: content/guides/kubernetes/what-is-velero.md
Comment (BrianMMcClain) [inline content/guides/kubernetes/what-is-velero.md:27]:
```suggestion
# Velero Components and Architecture
```

File: content/guides/containers/managing-container-images-registry.md
Comment (BrianMMcClain) [inline content/guides/containers/managing-container-images-registry.md:59]: Might be worth considering putting this in a "diagrams" subdirectory to add padding since it's pretty close to the edge of the image. Not a big deal if not though.

**Review by BrianMMcClain — DISMISSED** [review]

**Review by BrianMMcClain — APPROVED** [review]

**Pull request merged**
